pkg/types/validation/installconfig.go

```diff
@@ -125,11 +125,17 @@ func ValidateInstallConfig(c *types.InstallConfig, usingAgentMethod bool) field.
 	} else {
 		allErrs = append(allErrs, field.Required(field.NewPath("controlPlane"), "controlPlane is required"))
 	}
+
 	multiArchEnabled := types.MultiArchFeatureGateEnabled(c.Platform.Name(), c.EnabledFeatureGates())
 	allErrs = append(allErrs, validateCompute(&c.Platform, c.ControlPlane, c.Compute, field.NewPath("compute"), multiArchEnabled)...)
-	if multiArchEnabled {
-		allErrs = append(allErrs, validateMultiReleasePayload(c.ControlPlane, c.Compute)...)
+
+	releaseArch, err := version.ReleaseArchitecture()
+	if err != nil {
+		allErrs = append(allErrs, field.InternalError(field.NewPath(""), err))
```

Comment:
Do we need an empty new path? wouldn't just field work

Reply:
Updated.